model/atmosphere/diffusion/src/icon4py/model/atmosphere/diffusion/diffusion.py

```diff
@@ -62,6 +63,12 @@
 from icon4py.model.common.settings import xp
 from icon4py.model.common.utils import gt4py_field_allocation as field_alloc
 
+from icon4py.model.common.orchestration.decorator import (
+    orchestrate,
+    build_compile_time_connectivities,
```

Comment:
As I wrote elsewhere, I think the `build_compile_time_connectivities` should probably be somewhere in `common/grid/utils.py` or so. It is not anything dace specific.